src/schema/partner.ts

```diff
@@ -278,6 +278,15 @@ const PartnerType = new GraphQLObjectType({
           return exceptions[type] || type
         },
       },
+      website: {
+        description: "The gallery partner's web address",
+        type: GraphQLString,
+        resolve: root => {
+          if (root.website) {
+            return root.website || ""
```

Comment:
I don't think the `|| ""`  guard is strictly needed here? (The field is not declared to be non-nullable.)

In that case you can do away with the resolver completely, since the [default field resolver](https://github.com/artsy/metaphysics/pull/1362#discussion_r229079660) will simply expose the same-named field on the object in question.

Reply:
Yep, just caught that myself...just made the change but looks like CI tests are still running.